lib/LedgerSMB/Scripts/recon.pm

```diff
@@ -235,10 +236,10 @@ sub _display_report {
     for my $field (qw/ total_cleared_credits total_cleared_debits total_uncleared_credits total_uncleared_debits /) {
       $recon->{"$field"} = LedgerSMB::PGNumber->from_input(0);
     }
-    my $neg_factor = 1;
+    my $is_positive = 1;
     if ($recon->{account_info}->{category} =~ /(A|E)/){
```

Comment:
It's correct that positive assets (and expenses) are accounted/stored as negative values.